src/bundle/Resources/config/services/forms.yml

```diff
@@ -33,6 +33,11 @@ services:
 
     EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\ContentTypeChoiceLoader: ~
 
+    EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader\AvailableLocaleChoiceLoader:
+        arguments:
+            $availableTranslations: '%available_translations%'
+            $additionalTranslations: '$user_preferences.additional_translations$'
```

Comment:
Until we resolve race when building container and config resolver, it's not safe to use dynamic parameters directly in DI config. Have you tried to override default one by SA config? Sometimes you'll end up with default one anyway.